modules/gnome-extensions/README.md

```diff
@@ -0,0 +1,30 @@
+# `gnome-extensions`
+
+The `gnome-extensions` module can be used to install Gnome extensions inside system directory.
+
+This module is universally compatible with all distributions which ship Gnome, as it's not tied to specific distribution packaging format for installing extensions.  
+
+Almost all Gnome extensions are compatible for installation.  
+Only rare intervention that might be needed is for extensions which require some additional system dependencies, like Pano.   
+
+Thanks to https://extensions.gnome.org which provides releases of extensions as zips, it is very easy to maintain this module configuration.  
+The only maintenance is to bump the extension version when new Fedora/Gnome releases (around every 6 months).
+
+What does this module do?
+- It parses the gettext-domain that you inputted, along with the extension version
+- It downloads the extension directly from https://extensions.gnome.org
+- Downloaded extension archive is then extracted to temporary directory
+- All of its extracted files are copied to the appropriate final directories  
+  (`/usr/share/gnome-shell/extensions`, `/usr/share/glib-2.0/schemas`, & `/usr/share/locale`)
+
+# Usage
+
+To use this module, you need to input gettext-domain of the extension without @ symbol + the version of the extension in `.v%VERSION%` format in module recipe.  
```

Comment:
The string `%VERSION%` should not be used to represent the extension version unless it is actually what the user should input in the recipe to get some sort of dynamic version. This is because the `rpm-ostree` module uses `%OS_VERSION%` as a shorthand for the current Fedora version in repo URLs, and using the same syntax to mean different things might be confusing.

Reply:
I think that this part of the section can be removed with your latest doc changes, as everything necessary is explained in install instructions.